docs/guide-ru/start-workflow.md

```diff
@@ -37,15 +37,15 @@
 
 ```js
```

Comment:
Instead of changing apostrophe, we can remove js here to turn off highlighting. It's not JS anyway.

Reply:
Should I try to find all _```js_  values and replace them in all possible languages?

Reply:
Yes. Where appropriate.